From ef0de57423e239c3d9340496ada11c63cc25c243 Mon Sep 17 00:00:00 2001
From: Gaming4JC <g4jc@hyperbola.info>
Date: Sat, 3 Jun 2023 21:42:02 -0400
Subject: [PATCH 5/7] Disable SSLKEYLOGFILE in NSS

Useful for debugging, but also strips SSL on the local host which is bad for privacy.
https://firefox-source-docs.mozilla.org/security/nss/legacy/key_log_format/index.html
---
 security/nss/lib/ssl/Makefile | 2 +-
 security/nss/lib/ssl/ssl.gyp  | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/security/nss/lib/ssl/Makefile b/security/nss/lib/ssl/Makefile
index 8a8b06f4b5..9c273b9b2f 100644
--- a/security/nss/lib/ssl/Makefile
+++ b/security/nss/lib/ssl/Makefile
@@ -43,7 +43,7 @@ endif
 # Logging still needs to be enabled at runtime through env vars.
 NSS_ALLOW_SSLKEYLOGFILE ?= $(if $(BUILD_OPT),0,1)
 ifeq (1,$(NSS_ALLOW_SSLKEYLOGFILE))
-DEFINES += -DNSS_ALLOW_SSLKEYLOGFILE=1
+DEFINES += -DNSS_ALLOW_SSLKEYLOGFILE=0
 endif
 
 #######################################################################
diff --git a/security/nss/lib/ssl/ssl.gyp b/security/nss/lib/ssl/ssl.gyp
index 2aa35cc963..4719c3f24e 100644
--- a/security/nss/lib/ssl/ssl.gyp
+++ b/security/nss/lib/ssl/ssl.gyp
@@ -72,7 +72,7 @@
             'UNSAFE_FUZZER_MODE',
           ],
         }],
-        [ 'enable_sslkeylogfile==1', {
+        [ 'enable_sslkeylogfile==0', {
           'defines': [
             'NSS_ALLOW_SSLKEYLOGFILE',
           ],
-- 
2.41.0

